examples/lock-app/nrfconnect/README.md

```diff
@@ -101,135 +100,116 @@ the nRF52840's NFC antenna. Moreover in the reaction on sensing field from the
-`/var/chip/examples/lock-app/nrfconnect/build/zephyr/zephyr.hex`. Note that
-other operations described in this document like flashing or debugging can also
-be done in the container.
+Proceed with the [Building](#building) instruction.
 
 ### Using native shell
 
-Before building the example,
-[download the nRF Connect SDK and install all requirements](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html).
-Please read the
-[Supported nRF Connect SDK versions](#supported-nrf-connect-sdk-versions)
-section to learn which version to use to avoid unexpected compatibility issues.
+Follow the
+[guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_assistant.html#)
+to install nRF Connect SDK v1.4.0. Since further in the text we use command-line
+tools to build the example, installing SEGGER Embedded Studio is not required.
 
-If you don't want to use SEGGER Embedded Studio, you may skip the part about
-installing and configuring it.
+In case you have an older version of the SDK installed, use the following
+commands to update it to the recommended version. Replace `<nrfconnect-dir>`
+with a path to the nRF Connect SDK installation directory.
+
+        $ cd <nrfconnect-dir>/nrf
+        $ git fetch origin
+        $ git checkout v1.4.0
+        $ west update
 
 Download and install the
 [nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
 
 Download and install [GN meta-build system](https://gn.googlesource.com/gn/).
 
-Make sure that you source the following file:
-
-        $ source <ncs-dir>/zephyr/zephyr-env.sh
-
-> **Note:**
->
-> Ensure that `$ZEPHYR_BASE`, `$GNUARMEMB_TOOLCHAIN_PATH`, and
-> `$ZEPHYR_TOOLCHAIN_VARIANT` environment variables are set in your current
-> terminal before building. `$GNUARMEMB_TOOLCHAIN_PATH` and
-> `$ZEPHYR_TOOLCHAIN_VARIANT` must be set manually.
-
-After your environment is set up, you are ready to build the example. The
-recommended tool for building and flashing the device is
-[west](https://docs.zephyrproject.org/latest/guides/west/).
+Initialize environment variables referred to by CHIP and nRF Connect SDK build
+scripts. Replace `<nrfconnect-dir>` with a path to the nRF Connect SDK
+installation directory. Also replace `<toolchain-dir>` with a path to GNU Arm
+Embedded Toolchain.
 
-The following commands will build the `lock-app` example:
+        $ source <nrfconnect-dir>/zephyr/zephyr-env.sh
+        $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+        $ export GNUARMEMB_TOOLCHAIN_PATH=<toolchain-dir>
 
-        $ cd ~/connectedhomeip/examples/lock-app/nrfconnect
+## Building
 
-        # If this is a first time build or if `build` directory was deleted
-        $ west build -b nrf52840dk_nrf52840
+Navigate to the example's directory:
 
-        # Any subsequent build
-        $ west build
+        $ cd examples/lock-app/nrfconnect
 
-After a successful build, the binary will be available under
```

Comment:
I would not remove it, as it's quite useful for developer to know where to find build output file. I guess that you wanted to simplify instruction, but maybe it would be better to write something like: "West tool is able to find proper file and flash it on its own, but if you want, you can find output file there ..."?